manager/init.py

```diff
@@ -27,93 +15,42 @@ def __init__(self, sim_id=None):# {{{
         self.project_id=self.conf['project_id']
         self.scenario_id=self.conf['scenario_id']
         self.p=Psql()
-        self._clear_srv_anims()
-        self.s=Sqlite("{}/aamks.sqlite".format(os.environ['AAMKS_PROJECT']), 2)
-        if os.environ['AAMKS_WORKER'] == 'slurm':
-            new_sql_path = os.path.join(os.environ['AAMKS_PROJECT'], f"aamks_{sim_id}.sqlite")
-            shutil.copy(os.path.join(os.environ['AAMKS_PROJECT'], "aamks.sqlite"), new_sql_path)
-            self.s=Sqlite(new_sql_path)
-        self._clear_sqlite()
-        self.irange = SimIterations(self.project_id, self.scenario_id, self.conf['number_of_simulations']).get()
-        
+        if not sim_id:
+            si = SimIterations(self.project_id, self.scenario_id, self.conf['number_of_simulations'])
+            self.irange = si.get()
+            si.insert(self.conf['number_of_simulations'])
+        else:
+            self.irange = [sim_id, sim_id+1]
```

Comment:
We need `start` and `stop`  to be passed to `range()` from `self.irange`

```suggestion
            self.irange = [sim_id, sim_id+self.conf['number_of_simulations']]
```

Reply:
When `sim_id` is not `None` we perform only one particular iteration passed to worker with `OnInit()`. This is ok